release-4.1 - HTTPS required for repo1.maven.org (...) hibernate-entitymanager

## Observed behaviour
I have this error on the `release-4.1` branch and the tag `v4.1.0-rc.5`
```
[INFO] commons-entity-jpa 4.1.0-SNAPSHOT .................. FAILURE [  0.755 s]
[INFO] asqatasun-api 4.1.0-SNAPSHOT ....................... SKIPPED
(...)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project asqatasun-commons-entity-jpa: 
      Could not resolve dependencies for project org.asqatasun:asqatasun-commons-entity-jpa:jar:4.1.0-SNAPSHOT: 
      Failed to collect dependencies at org.hibernate:hibernate-entitymanager:jar:4.3.5.Final: 
      Failed to read artifact descriptor for org.hibernate:hibernate-entitymanager:jar:4.3.5.Final: 
      Could not transfer artifact org.hibernate:hibernate-entitymanager:pom:4.3.5.Final from/to central (http://repo1.maven.org/maven2/): 
      Failed to transfer file: http://repo1.maven.org/maven2/org/hibernate/hibernate-entitymanager/4.3.5.Final/hibernate-entitymanager-4.3.5.Final.pom. 
      Return code is: 501 , ReasonPhrase: HTTPS Required. -> [Help 1]
```

## Expected behaviour
compilation without error.

## Steps to reproduce the behaviour
```bash
cd ~ 
mv -v .m2 .m2_backup

git clone ...
cd Asqatasun
git checkout release-4.1
mvn clean install
```

## Your configuration
```bash
openjdk 11.0.6 2020-01-14
OpenJDK Runtime Environment (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1, mixed mode, sharing)

Apache Maven 3.6.0
Maven home: /usr/share/maven
Java version: 11.0.6, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
Default locale: fr_FR, platform encoding: UTF-8
OS name: "linux", version: "5.3.0-45-generic", arch: "amd64", family: "unix"
```

and also with jdk8:  https://github.com/Asqatasun/asqa.mvn/blob/master/Asqatasun/jdk-8/Dockerfile


## Solution

- Update [Pom.xml file (ligne 138) on the `release-4.1` branch](https://github.com/Asqatasun/Asqatasun/blob/release-4.1/pom.xml#L138) .
- see also [Pom.xml file (ligne 251) on the `develop` branch](https://github.com/Asqatasun/Asqatasun/blob/develop/pom.xml#L251), already corrected. .

--- a/pom.xml
+++ b/pom.xml
@@ -135,7 +135,7 @@
     </properties>
     <repositories>
         <repository>
-            <url>http://repo1.maven.org/maven2/</url>
+            <url>https://repo1.maven.org/maven2/</url>
             <id>central</id>
             <releases>
                 <enabled>true</enabled>

